src/views/conference/2018/index/index.scss

```diff
@@ -8,7 +8,7 @@
 .title-banner-image.mod-2018 {
     opacity: .75;
     margin-bottom: 1.75rem;
-    background-image: url("/images/conference/index/2017/title-banner.jpg");
+    background-image: url("/images/conference/index/2018/title-banner.jpg");
```

Comment:
I think you cropped the original image a little too much. With the current settings the sides of the image will be hidden if the aspect ratio is not as wide as the image. That's going to look a bit odd if just the nose of the person on the left is visible. I would crop the image with more of the table showing, say down to the top of the LEGO container in the foreground. Alternatively, you could keep the original image and just use the background position to keep the focus on the top half. The current background-position is `center` or `50% 50%`. You could change it to something like `50% 25%`.

Reply:
I Argge With @chrisgarrity